File: task/src/main/java/org/kharitonov/springWeb/config/SpringConfig.java
package org.kharitonov.springWeb.config;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.json.JsonMapper;
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
import org.kharitonov.springWeb.annotations.IdAnnotation;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class SpringConfig {

    @Bean(name = "objectMapper")
    public ObjectMapper objectMapper() {
        return JsonMapper.builder()
                .addModule(new JavaTimeModule())
                .build();
    }

    @Bean(name = "idAnnotationImpl")
    public IdAnnotation idAnnotation() {
        return new IdAnnotation();
    }
}

File: laba_chetyre/src/main/kotlin/by/bsut/labachetyre/config/SpringConfig.kt
package by.bsut.labachetyre.config

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.databind.json.JsonMapper
import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
import by.bsut.labachetyre.annotations.IdAnnotation
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class SpringConfig {

    @Bean(name = "objectMapper")
    fun objectMapper(): ObjectMapper = JsonMapper.builder().addModule(JavaTimeModule()).build()

    @Bean(name = "idAnnotationImpl")
    fun idAnnotation(): IdAnnotation = IdAnnotation()
}